src/Generator.TemplateProject/Custom/DateOnlyConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Generator.CustomModels;
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Xml;
+//using System.Runtime;
```

Comment:
I'm sorry - I missed this in the last PR. Otherwise everything looks great!

Reply:
Done